lib/css/exports/_stacks-constants-colors.less

```diff
@@ -419,6 +445,20 @@ body,
 
     // Scrollbars
     --scrollbar: rgba(255, 255, 255, .2);
+
+    // Syntax highlighting
+    --highlight-bg: #1c1b1b;
+    --highlight-color: @white; // AAA 17.19
```

Comment:
Having these stats available to point to is awesome! Do we want to add a little blurb to the `Code block` docs page saying something like

> All our colors are designed against WCAG best practices and have a contrast value of at least blah blah blah

Reply:
Done!